src/middleware/logging.rs

```diff
@@ -100,16 +122,16 @@ pub fn extract_context<B>(request: &Request<B>) -> Context {
 }
 
 /// Get TraceId from given [`Span`]
-pub fn get_span_trace_id(span: &Span) -> TraceId {
+pub fn get_span_trace_and_span_ids(span: &Span) -> (TraceId, SpanId) {
```

Comment:
maybe we should rename this fn to get_span_and_trace_ids ?

Reply:
Probably, yes. From where this value takes user can see in type of function argument